modules/platform/nuxeo-platform-video/nuxeo-platform-video-rest/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/management/VideosObject.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Charles Boidot
+ */
+package org.nuxeo.ecm.restapi.server.jaxrs.management;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static org.nuxeo.ecm.core.api.security.SecurityConstants.SYSTEM_USERNAME;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.ws.rs.FormParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+
+import org.apache.commons.lang3.StringUtils;
+import org.nuxeo.ecm.core.bulk.BulkService;
+import org.nuxeo.ecm.core.bulk.message.BulkCommand;
+import org.nuxeo.ecm.core.bulk.message.BulkStatus;
+import org.nuxeo.ecm.platform.video.service.RecomputeVideoConversionsAction;
+import org.nuxeo.ecm.webengine.model.WebObject;
+import org.nuxeo.ecm.webengine.model.impl.AbstractResource;
+import org.nuxeo.ecm.webengine.model.impl.ResourceTypeImpl;
+import org.nuxeo.runtime.api.Framework;
+
+/**
+ * since 11.4
+ */
+@WebObject(type = ManagementObject.MANAGEMENT_OBJECT_PREFIX + "videos")
+@Produces(APPLICATION_JSON)
+public class VideosObject extends AbstractResource<ResourceTypeImpl> {
+
+    // By default we only recompute renditions for document without any renditions
+    public static final String VIDEOS_DEFAULT_QUERY = "SELECT * FROM Document WHERE ecm:mixinType = 'Video' AND ecm:isProxy = 0 AND ecm:isVersion = 0";
+
+    /**
+     * @TODO Default query should only recompute rendition for videos with no recomputed renditions
+     *       (vid:transcodedVideos is NULL)
+     */
```

Comment:
Can you mode the TODO on the query and write it as a regular comment (not a javadoc one).